py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/convert_gtfs_rt.py

```diff
@@ -55,6 +55,11 @@ def __init__(self, config_type: ConfigType) -> None:
         else:
             raise NoImplException(f"No Specialization for {config_type}")
 
+        # get the start of the current hour. any files written after this will
+        # not be ingested until the next go around.
+        now = datetime.now(tz=timezone.utc)
+        self.start_of_hour = now.replace(minute=0, second=0, microsecond=0)
+
```

Comment:
at a start of hour member that we'll use to filter out files that are too recent.